components/post_emoji/post_emoji.tsx

```diff
@@ -1,24 +1,21 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-import React from 'react';
+import React, {CSSProperties} from 'react';
 import PropTypes from 'prop-types';
```

Comment:
@AGMETEOR you have the same `PropTypes` tests failing on this PR as in https://github.com/mattermost/mattermost-webapp/pull/3882. Try removing this line.